seqr/views/utils/test_utils.py

```diff
@@ -562,17 +562,37 @@ def assert_no_extra_anvil_calls(self):
         self.mock_get_group_members.assert_not_called()
 
 
-MOCK_AIRFLOW_URL = 'http://testairflowserver'
-DAG_NAME = 'LOADING_PIPELINE'
-PROJECT_GUID = 'R0001_1kg'
 
+PROJECT_GUID = 'R0001_1kg'
 
 class AirflowTestCase(AnvilAuthenticationTestCase):
+    MOCK_AIRFLOW_URL = 'http://testairflowserver'
     ADDITIONAL_REQUEST_COUNT = 0
+    DAG_NAME = 'LOADING_PIPELINE'
 
-    def setUp(self):
-        self._dag_url = f'{MOCK_AIRFLOW_URL}/api/v1/dags/{DAG_NAME}'
+    def setUp(self, **kwargs):
```

Comment:
`setUp` should not take keyword args, since you can definitionally set default values for kwargs in the functions that use them just make sure to set one such that a call to `setUp` with no kwargs does the first initialization you want